### DISCUSSION TITLE
设置禁止定位启动不了

### DISCUSSION THREAD

**Discussion by nozomileng** [XTLS/Xray-core#358] [Q&A] [answered]
文件设置好了不能启动，删除定位那行就能启动
{
    "inbounds": [ 
	{
      "port": ,
      "listen":"127.0.0.1",
      "protocol": "vless",
      "settings": {
        "decryption": "none",
        "clients": [
          {
            "id": "",
            "level": 0
          }
        ]
      },
      "streamSettings": {
        "network": "ws",
        "wsSettings": {
        "path": ""
        }
      }
    }],
    "outbounds": [ 
	{
            "protocol": "blackhole",
            "settings": {},
            "tag": "block"
        },
        {
            "protocol": "freedom",
            "settings": {},
            "tag": "direct"
    }
    ],
    "routing": {
        "domainStrategy": "IPOnDemand",
        "rules": [{
                "domain": [
	                    "regexp:(api|ps|sv|offnavi|newvector|ulog\.imap|newloc)(\.map|)\.(baidu|n\.shifen)\.com"
                	],
                "type": "field",
                "outboundTag": "block"
		}]
		}
}

**Comment by AkinoKaede** [comment, marked as answer]
本人测试，填写 port 和 uuid 后启动并无错误输出

**Reply by ghost** [reply]
同 @AkinoKaede 的结果，请尝试配置 [日志](https://xtls.github.io/config/base/log/)。

**Reply by nozomileng** [reply]
已经解决，多打了个逗号，并且下面的条例漏了/
